pkg/config/generate.go

```diff
@@ -29,7 +29,7 @@ func selectNodeImage(cs *acsapi.ContainerService) {
 	case "":
 		c.ImageSKU = "osa_" + strings.Replace(cs.Properties.OrchestratorProfile.OrchestratorVersion[1:], ".", "", -1)
 	case "centos7":
-		c.ImageSKU = "origin_" + strings.Replace(cs.Properties.OrchestratorProfile.OrchestratorVersion[1:], ".", "", -1)
+		c.ImageSKU = "okd_" + strings.Replace(cs.Properties.OrchestratorProfile.OrchestratorVersion[1:], ".", "", -1)
```

Comment:
I don't think we're supposed to be using OKD in any packaging.  What about oso (openshift origin)?.  I'm not in love with anything but origin, I want it back.

Reply:
If you'd prefer `oso' I can update it.  This process will cause another publish and is generally a 1-2 day process.  If we want to move forward with oso please let me know and I"ll kick that off.

Currently I'm awaiting to hear back on resurrecting `origin_`.

Reply:
I'd rather just get origin back.  :smile:

Reply:
I agree 100%.  In the meantime this is broken for a few of us and I suspect will break for others when origin_ unavailability propagates.

Reply:
if it's temporary then...

/lgtm

Reply:
> if it's temporary then...

Famous last words